<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EED_4_4D_Dialog</class>
 <widget class="QDialog" name="EED_4_4D_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1454</width>
    <height>1115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edge-Enhancing Diffusion based inpainting for 4D data</string>
  </property>
  <widget class="QLabel" name="label_eed44dInpaintImg">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>611</width>
     <height>601</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_eed44dInpaint">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>50</y>
     <width>400</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>Inputs</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_eed44dInpaintTol">
          <property name="text">
           <string>Tolerance</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_eed44dInpaintTol"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_eed44dInpaintTimeStep">
          <property name="text">
           <string>Time Step Size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_eed44dInpaintTimeStep"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_eed44dInpaintExplitScheme">
        <property name="text">
         <string>Explicit Scheme</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_eed44dInpaintFED">
        <property name="text">
         <string>Fast Explicit Diffusion (FED) Scheme</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_eed44dInpaintFSI">
        <property name="text">
         <string>Fast Semi_iterative Scheme (FSI)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_eed44dInpaintMSETol">
          <property name="text">
           <string>MSE Tolerance</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_eed44dInpaintMSETol"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_eed44dInpaintInnerFEDCycSize">
          <property name="text">
           <string>Inner Cycle Size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_eed44dInpaintInnerFEDCycSize"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_eed44dInpaintFSILoopSize">
          <property name="text">
           <string>Inner Loop Size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_eed44dInpaintFSILoopSoze"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_eed44dReferImgData">
        <property name="text">
         <string>Load Reference Image Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_eed44dInpaintUplImg">
        <property name="text">
         <string>Load Image Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_eed44dInpaintUplMask">
        <property name="text">
         <string>Load Mask Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_brainBinMaskLoad">
        <property name="text">
         <string>Load Brain Binary Mask</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_eed44dInpaintRun">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_eed44dInpaintMonitoring">
        <property name="text">
         <string>Monitoring</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_eed_brainBinMask">
        <property name="text">
         <string>with Brain Binary Mask</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_eed_brainBinMask_without">
        <property name="text">
         <string>without Brain Binary Mask</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_eed44dInpaintMaskImg">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>570</y>
     <width>450</width>
     <height>450</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>960</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_eed44dInpaintFileName">
      <property name="text">
       <string>File Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_eed44dInpaintFileName"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_eed44dInpaintSaveImg">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
